pyenvbuilder/commands/check.py

```diff
@@ -0,0 +1,87 @@
+'''
+
+Class for YAML file validation
+'''
+from codecs import open
+import errno
+from jsonschema import validate, exceptions
+import yaml
```

Comment:
`yaml` comes from `pyyaml` so it should be added as well to the `requirements.txt`.